src/components/pages/MyOrganisations/index.tsx

```diff
@@ -1,12 +1,72 @@
 import * as React from 'react';
+import { connect } from 'react-redux';
+import { history } from 'store';
 import styled from '@emotion/styled';
+import OrganisationCard from 'components/ui/OrganisationCard';
+import { fetchCurrentUserOrganisations } from 'store/actions/dashboard';
+import { getDateJoinedMessage } from '../../../helpers/dateHelpers';
 
 import __spacing from 'settings/__spacing';
 
-export default function MyOrganisations(): React.ReactElement<{}> {
-  return <MyOrganisations.Wrapper>My Organisations will go here.</MyOrganisations.Wrapper>;
+interface MyOrganisationsProps {
+  img: string;
+  name: string;
+  role: { name: string };
+  date: string;
+  isFetched: boolean;
+  organisations: object;
+  fetchCurrentUserOrganisations: () => void;
 }
 
+export function MyOrganisations(props: MyOrganisationsProps): React.ReactElement<{}> {
+  const { fetchCurrentUserOrganisations, organisations, isFetched } = props;
+  let organisationsToDisplay;
+
+  React.useEffect(function(): void {
+    fetchCurrentUserOrganisations();
+  }, []);
+
+  if (isFetched) {
+    try {
+      organisationsToDisplay = Object.values(organisations);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  if (isFetched) {
+    return (
+      <MyOrganisations.Wrapper>
+        {organisationsToDisplay.map(org => {
+          return (
+            <div key={org.organisation.id} onClick={() => history.push('/dashboard/profile', org)}>
+              <OrganisationCard
+                img={org.organisation.logoUrl}
+                name={org.organisation.name}
+                role={org.role.name}
+                date={getDateJoinedMessage(org.organisation.createdAt)}
+              />
+            </div>
+          );
+        })}
+      </MyOrganisations.Wrapper>
+    );
+  } else {
+    return <p> Sorry! you do not belong to any organisation yet </p>;
+  }
```

Comment:
ok, implementing it asap

Reply:
@chukwuemekachm I got feedback to add extra check showing when a user organisation is loading so that I can display `... Loading`, with that I think using ternary operation will almost be difficult. I am done implementing the feedback, however a solution on how to make it better will be nice. Thanks for the feedback